core/src/main/java/io/aiven/klaw/service/ClusterApiService.java

```diff
@@ -509,6 +509,9 @@ public ResponseEntity<ApiResponse> approveTopicRequests(
                   .schemaEnv(kwClustersSchemaEnv.getBootstrapServers())
                   .schemaEnvProtocol(kwClustersSchemaEnv.getProtocol())
                   .build();
+        } else {
+          clusterTopicRequest =
+              clusterTopicRequest.toBuilder().deleteAssociatedSchema(false).build();
```

Comment:
I think you can add this to the end of line 472 and it will be false for all except when it is overwritten because it is true?

Reply:
As the parameter is applicable only for delete scenarios, I didn't build it up there.